codegen/src/test/resources/software/amazon/awssdk/codegen/poet/model/eventone.java

```diff
@@ -21,24 +21,24 @@
 /**
  */
 @Generated("software.amazon.awssdk:codegen")
-public final class EventOne implements SdkPojo, Serializable, ToCopyableBuilder<EventOne.Builder, EventOne>, EventStream {
+public class EventOne implements SdkPojo, Serializable, ToCopyableBuilder<EventOne.Builder, EventOne>, EventStream {
```

Comment:
Can we add sdk internal api annotation and javadocs?

Reply:
I don't think it would make sense to make these internal because they're used as the return type of static builder methods on the event stream interface. Also seems like it's possible that an event shape can be used in a non event stream context, like a member on a non `eventstream` structure.